### PR TITLE
batch size is set to 2 to support models with BatchNorm

### PR DESCRIPTION
**Pull request by FlyHighest**
I have the same problem as issue #4 . This problem is caused by batch normalization, which requires batchsize>=2.

### CODE REVIEW

**Comment by sksq96** [comment]
Thanks for pointing this out.
